google/cloud/bigtable/examples/bigtable_samples_instance_admin_ext.cc

```diff
@@ -162,19 +174,22 @@ void CreateDevInstance(std::string project_id, int argc, char* argv[]) {
 
   std::cout << "\nCheck Instance exists:\n";
   auto instances = instance_admin.ListInstances();
+  if (!instances) {
+    throw std::runtime_error(instances.status().message());
+  }
   auto instance_name =
       instance_admin.project_name() + "/instances/" + instance_id.get();
   bool instance_exists =
-      instances.instances.end() !=
+      instances->instances.end() !=
       std::find_if(
-          instances.instances.begin(), instances.instances.end(),
+          instances->instances.begin(), instances->instances.end(),
           [&instance_name](google::bigtable::admin::v2::Instance const& i) {
             return i.name() == instance_name;
           });
-  if (!instances.failed_locations.empty()) {
+  if (!instances->failed_locations.empty()) {
     std::cerr
         << "The service tells us it has no information about these locations:";
-    for (std::string const& location : instances.failed_locations) {
+    for (std::string const& location : instances->failed_locations) {
       std::cout << " " << location << "\n";
```

Comment:
here too. I realize that this isn't code that you changed in this PR, but it's close. Do we want to change it now? If we'd rather do this in a separate PR, that's fine with me, FWIW.

Reply:
How about I send you another PR with those issues addressed today then?

Reply:
SGTM.

Reply:
Can I have you LGTM on this PR then to submit it?